lib/serverengine/socket_manager_unix.rb

```diff
@@ -80,7 +80,8 @@ def stop_server
         @tcp_sockets.reject! {|key,lsock| lsock.close; true }
         @udp_sockets.reject! {|key,usock| usock.close; true }
         @server.close unless @server.closed?
-        @thread.join
+        # Ruby 2.1 has a problem around thread join
```

Comment:
Could you write more detailed information?
What happens, hang,raise an exception or etc?